pkg/util/oc/pod_commands.go

```diff
@@ -94,10 +94,33 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 func WaitCondition(t test.TestHelper, ns string, kind string, name string, condition string) {
 	t.T().Helper()
 	retry.UntilSuccessWithOptions(t, retry.Options().MaxAttempts(30), func(t test.TestHelper) {
-		shell.Executef(t,
-			fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s"),
+		cmd := fmt.Sprintf(`oc wait -n %s %s/%s --for condition=%s  --timeout %s`, ns, kind, name, condition, "10s")
+		shell.Execute(t,
+			cmd,
 			assert.OutputContains(condition,
 				fmt.Sprintf("Condition %s met by %s %s/%s", condition, kind, ns, name),
 				fmt.Sprintf("Condition %s not met %s %s/%s, retrying", condition, kind, ns, name)))
 	})
 }
+
+func DeletePod(t test.TestHelper, podLocator PodLocatorFunc) {
+	t.T().Helper()
+	var pod NamespacedName
+	retry.UntilSuccess(t, func(t test.TestHelper) {
+		pod = podLocator(t)
+	})
+	cmd := fmt.Sprintf(`oc delete pod %s -n %s`, pod.Name, pod.Namespace)
+	shell.Execute(t,
+		cmd,
+		assert.OutputContains("deleted",
+			fmt.Sprintf("Pod %s is being deleted", pod.Name),
+			fmt.Sprintf("Pod %s deletion return an error", pod.Name)))
```

Comment:
This should be in the retry above. Also `cmd` should be inlined, as there's no benefit of having the command in a variable.

Reply:
Done